util/io/files/tar.go

```diff
@@ -73,30 +72,30 @@ func Untgz(dstPath string, r io.Reader) error {
 		}
 
 		target := filepath.Join(dstPath, header.Name)
-
 		// Sanity check to protect against zip-slip
-		if !strings.HasPrefix(target, filepath.Clean(dstPath)+string(os.PathSeparator)) {
+		if !Inbound(target, dstPath) {
 			return fmt.Errorf("illegal filepath in archive: %s", target)
 		}
 
 		switch header.Typeflag {
 		case tar.TypeDir:
-			err := os.MkdirAll(target, 0755)
+			err := os.MkdirAll(target, 0755) // lgtm[go/zipslip]
```

Comment:
As I understand code scanning does not really respect `// lgtm[go/zipslip]`, right? Please remove it if it is not used.

Reply:
I don't understand why it doesn't. I opened an issue with codeql-go project to understand this better:
https://github.com/github/codeql-go/issues/703

Removing this for now.